### PR TITLE
Update styles for buttons on Graph and ListView

### PR DESCRIPTION
**Pull request by joseartrivera**
### Description of the changes:
- Updates graph button styles to be consistent/cleaner
- Updates margins on Equation ListView
- Updates Add New Equation button to be more like a sticky footer

### How changes were validated:
- Manual tests

### CODE REVIEW

File: src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp [hunk 1]
```diff
@@ -158,3 +159,8 @@ void EquationInputArea::ReloadAvailableColors(bool isHighContrast)
         equationViewModel->LineColor = AvailableColors->GetAt(m_lastLineColorIndex);
     }
 }
+
+void CalculatorApp::EquationInputArea::UserControl_SizeChanged(Platform::Object ^ sender, Windows::UI::Xaml::SizeChangedEventArgs ^ e)
+{
+    EquationInputList->MaxHeight = ActualHeight - 44;
```
Comment (joseartrivera) [inline src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp]: This is the best solution I could come up with, would love to hear of any alternatives. The goal is to get the button to add an equation to be directly below the ListViewItems until it needs to scroll, at that point it becomes a sticky footer.

Comment (rudyhuyn) [reply src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp]: I know a trick to do what you want! Let's chat about it tomorrow

Comment (rudyhuyn) [reply src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp]: You need 2 things:
- Grid with 2 rows, one * (for the listview) and one auto (for the footer) and set `VerticalAlignment` to _Top.
- this Grid must be hosted in a Panel having a restricted Height (Grid, Border, not StackPanel or ScrollViewer)

```xaml
<Grid> <! -- parent with a restricted height -->
   <Grid VerticalAlignment="Top">
            <Grid.RowDefinitions>
                <RowDefinition Height="*" />
                <RowDefinition Height="auto" />
            </Grid.RowDefinitions>
            <ListView x:Name="MyList">
                <ListView.ItemTemplate>
                    <DataTemplate>
                        <TextBlock FontSize="60" Text="HELLO" />
                    </DataTemplate>
                </ListView.ItemTemplate>
            </ListView>
            <TextBlock Grid.Row="1" FontSize="60" Text="Footer" />
        </Grid>
</Grid>
```

Comment (joseartrivera) [reply src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp]: This is great, thank you!

File: src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml [hunk 2]
```diff
@@ -9,48 +9,91 @@
...
         <ListView x:Name="EquationInputList"
-              VerticalContentAlignment="Top"
-              IsItemClickEnabled="False"
-              ItemsSource="{x:Bind Equations}"
-              SelectionMode="None">
+                  IsItemClickEnabled="False"
+                  ItemsSource="{x:Bind Equations}"
+                  SelectionMode="None">
+
+            <ListView.Style>
```
Comment (sanderl) [inline src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml]: Can you add a comment what this is doing? It looks like it is just empty and doing nothing.

File: src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml [hunk 3]
```diff
@@ -9,48 +9,91 @@
...
 
         <ListView x:Name="EquationInputList"
-              VerticalContentAlignment="Top"
-              IsItemClickEnabled="False"
-              ItemsSource="{x:Bind Equations}"
-              SelectionMode="None">
+                  IsItemClickEnabled="False"
+                  ItemsSource="{x:Bind Equations}"
+                  SelectionMode="None">
+
```
Comment (sanderl) [inline src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml]: nit: spacing

File: src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml (hunk 2 above)
Comment (sanderl) [reply src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml]:
```suggestion
<!-- Removes animations from the ListView Style. -->
            <ListView.Style>
```

**Pull request merged**